src/LAMMPS.jl

```diff
@@ -191,14 +194,11 @@ function Base.showerror(io::IO, err::LAMMPSError)
     print(io, "LAMMPSError: ", err.msg)
 end
 
-function check(lmp::LMP)
+function check(lmp::Union{LMP, Ptr{Cvoid}})
```

Comment:
Why the change to allow `void*`

Reply:
Same for LAMMPSError

Reply:
Look at the [Documentation](https://docs.lammps.org/Library_utility.html#_CPPv416lammps_has_errorPv) for `lammps_has_error` and `lammps_get_last_error_message`. They specifically allow the handle to be NULL in case we create a lammps instance with invalid command line arguments. However, because we check that our handle isn't Null before converting our LMP object to the Pointer, we'd get a confusing error message.

Reply:
I would then rather inline the error handling code. This ought to be the only case where we accept a raw pointer in the Julia API.

Reply:
My reasoning was that `check` was mostly for internal use anyways. I'll go ahead and inline the error handling then.